### ISSUE TITLE
Pop up context menus on touch events.

### ISSUE THREAD

**Issue by mbebenita** [wasdk/WebAssemblyStudio#72]
On mobile there's currently no way to bring up context menus. We should trigger these using a long press, or whatever makes sense.

**Comment by haxxorsid** [comment]
Maybe long press event should be added in DirectoryTree component, but I have a problem in accessing onContextMenu( which is written inside Controller in DirectoryTree) event when a long press is performed.

**Comment by mbebenita** [comment]
Yeah, there's some digging around in the VS Code sources for this. If you change the include path in `index.html` for the Monaco Editor from `monaco-editor/min/...` to `monaco-editor/dev/...` you can at least debug the sources.
